### DISCUSSION TITLE
news/20241126/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#440] [Website Discussions]
# news/20241126/

Strada & Michael Cioni have just released the full video of Michael's presentation from the 2024 Final Cut Pro Creative Summit!

https://fcp.cafe/news/20241126/

**Comment by 4MilePost** [comment]
I would love for this to do what it says it does, but...I cannot get the Media Sync Extension to authorize. I tried the free subscription. That did not work. I tried purchasing the app with the Lifetime Buy option, and that didn't work. Bummer.
